webpack.config.js

```diff
@@ -19,6 +19,14 @@ let rules = [
         [`@babel/preset-react`]
       ]
     }
+  },
+  {
+    test: /\.ftl$/,
+    use: [
+      {
+        loader: `raw-loader`
```

Comment:
is there no dedicated ftl loader that ensures that .ftl files are actually valid, rather than arbitrary string data? This should not silently pass if the ftl file is malformed...

Reply:
I don’t think there is, but I also don’t think it matters, because unlike gettext, Fluent is [resilient and has a strong recovery logic](https://github.com/projectfluent/fluent/wiki/Fluent-vs-gettext) so things shouldn’t break if a file is malformed. In our case, Fluent would just fall back to the strings in the "en" ftl file. Right @stasm?

Reply:
Also I think Pontoon cannot commit a malformed .ftl file

Reply:
Yes, that's correct.

Reply:
To expand on this: Fluent is designed such that a syntax error can only break the messages around it. Then, parsing recovers from the error and resumes at the next well-formed message.